src/TestSuite/TestCase.php

```diff
@@ -33,6 +33,7 @@
 abstract class TestCase extends BaseTestCase
 {
 
+    use EmailTrait;
```

Comment:
Having this trait be something people opt-into using would make it less magic, and there are lots of test cases where we don't need email assertions.

Reply:
Would you also see `TestEmailTransport::clearEmails();` as something the user should do on their own in `tearDown()`?

Reply:
Yeah I think so.